fhir-search/src/main/java/com/ibm/fhir/search/util/SearchUtil.java

```diff
@@ -758,13 +334,13 @@ private static SearchParameter getSearchParameterByUrlIfPresent(Map<String, Para
                     loggedValue = expression.getValue();
                 }
 
-                log.finest(String.format(EXTRACT_PARAMETERS_LOGGING, parameter.getCode().getValue(), loggedValue));
+                log.finest(String.format(EXTRACT_PARAMETERS_LOGGING, code, loggedValue));
```

Comment:
```suggestion
                log.finest(String.format(() -> EXTRACT_PARAMETERS_LOGGING, code, loggedValue));
```
slight optimization by deferring to a llambda (the trick here is those values must be final)

Reply:
its already behind a `log.isLoggable(Level.FINEST)`

Reply:
I totally missed the collapsed section